openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -21,6 +21,7 @@
 
 from ccx_keys.locator import CCXLocator
 
+THUMBNAIL_DIMENSIONS = (375, 200)
```

Comment:
In my small sampling, doing that about triples the file size, and my understanding is that we don't currently serve assets properly for retina anyway (though I'm really fuzzy on what that actually means). In the student dashboard, this is scaled down to 197x105 at the moment.

I'm not destroying the originals, making an extra set of thumbnails is something that we can always tack on later.